.vib/rails/goss/vars.yaml

```diff
@@ -0,0 +1,12 @@
+binaries:
+  - rails
+  - ruby
+directories:
+  - mode: "0755"
+    paths:
+      - /opt/bitnami/ruby/lib
+      - /opt/bitnami/ruby/bin
```

Comment:
I don't think we are modifying either of these folders' permissions. We are only doing it so for [/app](https://github.com/bitnami/containers/blob/main/bitnami/rails/7/debian-11/rootfs/opt/bitnami/scripts/rails/postunpack.sh), which should be included here.